examples/default_values_backend_app/README.md

```diff
@@ -0,0 +1,71 @@
+# Contentful Backend Demo APP
+
+A demo App that will prefill certain values when an entry is created.
+
+The purpose of this demo is to show how backend Apps can be setup and used.
+
+
+## Prerequisites
+This demo requires node.js, yarn, and some way of making the running server
+available to contentful. For local development it is easiest to use
+[ngrok](https://ngrok.com/)
+
```

Comment:
Maybe we could link to directly to a download page or setup instructions for `ngrok` for Mac, Linux or Windows? I bet a lot users won't have it installed right away. And maybe add some explanations of what is `ngrok` is doing?  Adding a screenshot of what to note down when running `ngrok` would also help. If I have never used ngrok I would be a little lost

Reply:
Why do we need ngrok at all here?

Reply:
@fabe this is for running the app locally - you need some way to make your localhost available for App events. Ngrok just seemed to be the easiest way to do this to me. If you have another suggestion I'd be happy to change it. Otherwise I'll add some more details here :)

Reply:
Cool, yeah including that in the readme would be great!

Reply:
I think we should mention that we need to use `https` for this url